### ISSUE TITLE
Show what info rageshake is going to send

### ISSUE THREAD

**Issue by 26000** [vector-im/riot-android#2183]
When I rageshake my phone and file a bug report, I'm asked to tick what I'm going to send. A screenshot, logs, crash logs. I don't know what's in the logs, I want to read them before sending! And the screenshot. Wanna make sure there are no bugs making my private rooms' screenshots sent instead of what I wanted.

The system is nice, but should be a little more transparent.
Thanks!

Sorry if this is a duplicate, couldn't find anything related.

**Comment by Qwertie-** [comment]
Agreed, it does feel a little uncomfortable sending an unknown amount of data from an app with sensitive data on it. 

Redhat's bug reporter "ABRT" does this well and allows you to edit out any sensitive info before sending.

**Comment by bmarty** [comment]
The screenshot is now shown in the BugReport screen (#2318 in v0.8.13). For the logs is not available yet. Normally they do not contains any personal/private data like message content for instance.

**Comment by aaronraimist** [comment]
Related iOS issue: 
https://github.com/vector-im/riot-ios/issues/1836

**Comment by aaronraimist** [comment]
![img_20181213_202909](https://user-images.githubusercontent.com/5855073/49947168-2c90f300-feb6-11e8-8cd9-bad85e021313.jpg)

`@imoteph:matrix.org` also mentioned that the "No" or "Yes" option on the initial rageshake screen is confusing. It looks like you are going to submit it right then and you wouldn't have a choice on what to send.

**Comment by bmarty** [comment]
Yes, I suggest to improve the wording, maybe:
`You seem to be shaking the phone in frustration. Would you like to open the bug report screen?`

**Comment by bmarty** [comment]
Ok, I've updated the wording

**Comment by bmarty** [comment]
Re-opened to track the preview of logs
